client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -581,8 +618,16 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { needsVerification, siteIsUnlaunched, translate } = this.props;
+		const { needsVerification, needsDomainVerification, siteIsUnlaunched, translate } = this.props;
```

Comment:
Should we not be importing `needsEmailVerification` instead of `needsVerification`?